__app__/hyper_func/__init__.py

```diff
@@ -0,0 +1,27 @@
+"""
+Importing Hyper.ag data using Azure FunctionApp.
+"""
+
+from datetime import datetime, timedelta, timezone
+import logging
+
+import azure.functions as func
+
+from __app__.crop.ingress_hyper import import_hyper_data
+from __app__.crop.constants import SQL_CONNECTION_STRING, SQL_DBNAME
+
+
+def hyper_import(mytimer: func.TimerRequest):
+    """
+    The main Hyper import Azure Function routine.
+    """
+
+    utc_timestamp = datetime.utcnow().replace(tzinfo=timezone.utc).isoformat()
+    logging.info("Python Hyper timer trigger function started at %s", utc_timestamp)
+
+    dt_to = datetime.now()

```

Comment:
I'm not 100% sure, but I think `now()` will use the local time zone by default. This contradicts the datetime formatting in [`get_api_sensor_data`](https://github.com/alan-turing-institute/CROP/pull/247/files#diff-7cb437aca0481cad28b8a0bfa968cc710f5e906e7eaedc6505bdc59b946dc07cR55) where the datetime value is formatted as UTC (`+ "Z"`). This means that we are not actually requesting the latest hour.

Reply:
Great spot - thanks Rizo!   Interestingly, when I run `datetime.now()` on my machine, it indeed gets local time, but when I run it on the docker container that the Azure function app is supposed to use, I get UTC (perhaps the docker container doesn't know where it is...)
Anyway, it is an easy and unambiguous fix for us to use `datetime.utcnow()`, so I will make that change now - thanks again!

Reply:
@nbarlowATI most cloud providers I worked with normally set the timezone to UTC by default in their instances. I'm certain this alone avoids some nasty bugs in production :)